main/webapp/modules/core/scripts/index/open-project-ui.js

```diff
@@ -199,7 +285,7 @@ Refine.OpenProjectUI.prototype._renderProjects = function(data) {
           
           return htmlDisplay;
       })() +     
-      '</tr></thead><tbody id="tableBody"></tbody></table>'
+      '</tr></thead><tbody id="tableBody"> </tbody></table>  <div id="no-results-message"> No Results </div>'
```

Comment:
Nice! We need to be able to translate this message with `i18n` (for instance see the `index.js` file you edited where we have things like `$("#or-index-pref").text($.i18n('core-index/preferences'));`).

Also, as we are normalizing the case of our messages at the moment (see #3363) we should make this be `No results` instead of `No Results`.